opentelemetry-stackdriver/src/lib.rs

```diff
@@ -311,6 +311,11 @@ where
                 }
             };
 
+            let parent_span_id = match span.parent_span_id {
+                SpanId::INVALID => "".to_owned(),
+                _ => hex::encode(span.parent_span_id.to_bytes()),
+            };
```

Comment:
Style nit: can just inline this into the initializer below, I think?

Is there documentation for this? Should we point to it here?